### DISCUSSION TITLE
Spanish translation

### DISCUSSION THREAD

**Discussion by Quarel** [Muttley/foundryvtt-fallout#556] [Ideas]
Hi, I'd like to help translate this system into Spanish. I don't know much about git hub, but I'm a fast learner. How could I help, if possible?

**Comment by QckSilverDragon** [comment]
All the translations are handled on Crowdin.com. You can request a new language there and Muttley can add it in for you to translate.

https://crowdin.com/project/foundryvtt-fallout

**Reply by guver** [reply]
hey. Can I also help?

**Reply by Muttley** [reply]
> hey. Can I also help?

Of course, just make an account on Crowdin and start adding translations: https://crowdin.com/project/foundryvtt-fallout

**Reply by guver** [reply]
Ok so now translations are at 100%.
Also reviewed all existing translation and fixed a bunch of them. Thanks to @Quarel for all the initial work.

**Comment by Muttley** [comment]
@Quarel Help with translations is always welcome.  Thanks. :)

I've added Spanish as a supported language to the Crowdin project @QckSilverDragon linked above, so once you sign up there you can just work your way through the strings.  Will require a new system release to be available in the interface though, but we can do that once you've had some time to work on them.

**Reply by Quarel** [reply]
Ok, thanks. I'm already working on it
